src/pages/GraduationCalculatorPage/components/GeneralCourse/GeneralCourseListModal/index.tsx

```diff
@@ -0,0 +1,39 @@
+import CloseIcon from 'assets/svg/close-icon-grey.svg';
+import SemesterList from 'pages/TimetablePage/components/SemesterList';
+import SemesterCourseTable from 'pages/GraduationCalculatorPage/components/CourseTable/SemesterCourseTable';
+import styles from './GeneralCourseListModal.module.scss';
+
+export interface GeneralCourseListModalProps {
+  frameId: number;
+  onClose: () => void;
+}
+
+function GeneralCourseListModal({
+  frameId,
+  onClose,
+}: GeneralCourseListModalProps) {
+  return (
+    <div className={styles.background}>
+      <div className={styles.container}>
+        <div className={styles.header}>
+          <p className={styles.header__title}>학기 교양 개설 목록</p>
+          {/* eslint-disable jsx-a11y/control-has-associated-label */}
+          <button
+            type="button"
+            onClick={onClose}
+            className={styles['header__close-button']}
+          >
+            <CloseIcon />
+          </button>
```

Comment:
여기서 disable한 오류는 button 태그의 내부에 텍스트가 없으면 생기는 오류예요. 지금 svg 컴포넌트만 들어갔기 때문에 발생했어요. 이때 disable 말고 
```suggestion
          <button
            type="button"
            onClick={onClose}
            className={styles['header__close-button']}
            aria-label="닫기 버튼"
          >
            <CloseIcon />
          </button>
```
이렇게 `aria-label`을 추가하면 없어져요!
+ aria 가 무엇을 뜻하는지 찾아보면 더 좋을 것 같아요~